src/sqs/error-handlers.ts

```diff
@@ -85,10 +86,16 @@ const maybeHandleNonRetryableResponseCode = <MessagePayload extends BaseMessageP
 
 const maybeHandleRateLimitingError = <MessagePayload extends BaseMessagePayload>(error: Error, context: SQSMessageContext<MessagePayload>): ErrorHandlingResult | undefined => {
 	if (error instanceof GithubClientRateLimitingError) {
+		const DELAY_BUFFER_STEP = 10;
```

Comment:
shouldn't it be among all the constants? And probably prefixed with `RATE_LIMIT_`?

Reply:
haha, i got stuck between two worlds, a lot of me wanted to bring the other ones inside here. moved with the rest!